### ISSUE TITLE
blower: ensure presence and functionality

### ISSUE THREAD

**Issue by Hurricos** [LaboratoryB/camfive#1]
The seller does not have a blower for us.

The blower is shown here: https://laboratoryb.github.io/camfive/docs/Laser_Machine_INSTALLATION_MANUAL.pdf#page=8

![image](https://github.com/user-attachments/assets/585e2943-c0c6-4767-94a1-bf1b903d37dc)

Jake may have access to a blower.

**Comment by heaventwig** [comment]
For temporary exhaust routing, add a T junction to exhaust fan duct (PVC) overhead in garage, adjacent to exit from garage, and run flex tubing from laser, via rafters, to the T junction. Should have louver-style air handling valves to prevent backflow into either pipe, as the tube draws from (and could return to) house air.

**Comment by heaventwig** [comment]
Tim says he's trying vent fans from Vevor, says they are much quieter than the "big bois" and rated for twice the CFM (a rating which Tim questions, but the quieterness is nice).

**Comment by DaBlender** [comment]
My in line blower. 4” inch connection, not a lot of duct but it’s not expensive. I have not had any issues with it. 
![6F70EF4A-EA2F-4D0B-B0A1-F1F3A16F22D0](https://github.com/user-attachments/assets/bd6034ae-30fd-4f95-aab3-bac4ce004b35)


I forget the tool name to measure flow but would be handy especially to any safety check

**Comment by heaventwig** [comment]
As I prep tubing for exhaust, I'm curious about reasons for and against including a dust collection system somewhere along the way before the output air (with all of its fumes and combustion byproducts) gets pushed into the world at large...

**Comment by DaBlender** [comment]
A common method for air filtration is to use a pre-filter, an activated
carbon filter, and a HEPA filter. This ensures that any unsafe particulates
are caught in the filter, and the air will be much safer to breathe. You
can either apply the filter to your laser cutter's exhaust or duct the air
into a fume coffin.

**Comment by Hurricos** [comment]
> A common method for air filtration is to use a pre-filter, an activated carbon filter, and a HEPA filter. This ensures that any unsafe particulates are caught in the filter, and the air will be much safer to breathe. You can either apply the filter to your laser cutter's exhaust or duct the air into a fume coffin.

([cited](https://monportlaser.com/blogs/how-to/how-to-vent-your-monport-laser-cutter))

I never paid attention to the exhaust filtering setup at Chatt*Lab (@ttsgeb?), but the consensus([^1][^2][^3][^4]) seems to be that if you cut certain materials, lasers make nasty fumes, AKA "LGACs" (Laser Generated Air Contaminants). LGACs are primarily a concern when cutting certain metals or plastics[^5]; if those types of materials are on the menu, and you're going to be cutting them in quantity, definitely get a filter.

[^1]: https://www.instructables.com/Fume-Coffin-Laser-Cutter-Exhaust-Vent-Filter/

[^2]: https://www.osti.gov/servlets/purl/1959231

[^3]: https://community.glowforge.com/t/fume-coffin/8848/2

[^4]: https://www.tlm-laser.com/info-centre/news-and-events/lasers-fumes-and-health

[^5]: https://ehs.oregonstate.edu/sites/ehs.oregonstate.edu/files/pdf/si/lasercuttersi.pdf

**Comment by heaventwig** [comment]
Okay, so I definitely want to construct an exhaust vent filtration system. Next questions / todos are around defining specs for that device and system.
- [x] define/identify CFM flow specification for exhaust ventilation from 150W CO<sub>2</sub> laser cutter

**Comment by DaBlender** [comment]
820m^3/h. Per manual.

The coffin method is cool. Beware front filter being anything that melts.

**Comment by heaventwig** [comment]
> Beware front filter being anything that melts.

In this I read "leave ample room for exhaust to cool before it hits anything temperature-sensitive."

**Comment by heaventwig** [comment]
also noting here, for future reference, that 820m<sup>3</sup>/h is just under 490 CFM (cubic feet per minute).

**Comment by DaBlender** [comment]
I’ve seen paper cinder and fly into the vent, melting a hepa. Metal grid
went before.

**Comment by heaventwig** [comment]
Adding to concept plan: metal wire mesh grid pre-prefilter.

**Comment by heaventwig** [comment]
ventilation plan:

1. metal wire mesh pre-prefilter on camfive air outlets
2. 6" flex tubing to [fume coffin](https://www.instructables.com/Fume-Coffin-Laser-Cutter-Exhaust-Vent-Filter/) (eventually install [Pre-Filter Addendum](https://www.instructables.com/Fume-Coffin-Pre-Filter/)) — need to redesign for higher CFM throughput; this design is specced for 100CFM and we need more like 500. 
   - alternatively, use premade air scrubber such as [this VEVOR unit](https://www.vevor.com/air-scrubber-c_12068/vevor-air-scrubber-with-3-stage-filtration-stackable-negative-air-machine-800-cfm-air-cleaner-with-merv10-carbon-h13-hepa-for-home-industrial-and-commercial-use-p_010226230633), which has 15.5" intake and 8.5" outlet, for which we'd need to supply some size adjustments; note $350 price on this unit in 2024 is pretty close to the total BOM for the above Instructable for the fume coffin, which was priced when posted over seven years ago. (Note also that although this unit advertises 800CFM that's likely assuming no other interference with pressure or air supply.)
3. 6" flex tubing from blower output on fume coffin up to right-angle turn PVC pipe in rafters; simultaneously or in-line with right-angle joint, step down to 4" PVC
4. run 4" PVC to in-line 4" VEVOR blower (plug in adjacent to garage door opener)
5. run 4" PVC to 90º turn with valve flaps (to prevent backdraft from other devices on same exhaust pipe)
6. T-join (or angle-in join) PVC connection with existing exhaust vent, with added valve flaps (to prevent backdraft into conditioned interior home air from laser exhaust)

**Comment by DaBlender** [comment]
I’d rather metal than PVC for heat reasons. This is an hvac kinda thing
maybe pvc is ok just check?
 Not sure I’ve seen a mix with plumbing. No idea if that passes code.


And yes flex tubing but metal. Sounds like you figured it all out!  In a
commercial building I’d defer to certified hvac.

**Comment by heaventwig** [comment]
with the VEVOR filter, we might:
1. rig a metal ducting box for the 15.5" intake (call that BOX A),
2. provide BOX A with two 6-inch inlets and a metal wire mesh pre-filter,
3. run [6-inch flexible metal ducts](https://www.lowes.com/pd/Lambro-6-in-x-96-in-Aluminum-Semi-rigid-Flexible-Duct/3240913) from the camfive air outlets to BOX A
4. rig BOX B on the 8.5" air outlet from the VEVOR device, with 4" outlet
5. run 4" flexible ducting from BOX B to rafters
6. join flexible duct to 90º PVC at rafters, all at 4" diameter
7. continue ceiling rig as described in https://github.com/LaboratoryB/camfive/issues/1#issuecomment-2368545286 from point 4 onward.

**Comment by ttsgeb** [comment]
It's worth noting that the inline blower I'm using is an 8'' one, and it does struggle to keep up with heavy usage.

Chattlab doesn't have any filtration at all, they just pump it all into the atmosphere and this is definitely fine and not a major concern that everyone refuses to address.

**Comment by DaBlender** [comment]
Does the flow not match the machines?

I hate bringing up Generator, but that is my experience and they just pump
it right to the outside. They did internal filtering for a while, but
didn’t want to change HEPA filters so that was thousands of dollars gone.
Plus it was in downtown Burlington and we got so many complaints from
people cutting acrylic and rubber and all kinds of stuff.  I don’t think
there’s any actual danger when it gets that dispersed into the air.
Definitely not more than the factories were surrounded by perhaps just
taking a walk along the highway is worse  but you expect smells there and
your neighbors do not and I don’t want anyone to get in trouble   it’s
simple enough to do upfront to ensure that everyone is safe and on board.
Something about taking reasonable measures or something.  Despite the flow
rate of the fan, it’s going to be a little bit different with all the
different diameters in the filtration‘s so I personally want to know what
the rate is at the end because I like efficiency.  And I certainly don’t
wanna breathe any of this, and I’ve done that far too much by the
inadequate ventilation at Generator

They actually hired a friend who knew what they were doing so they said and
they used plastic 4 inch duct, which ended up with holes melted in it, and
no one Iistened.

I guess I’ll be devils advocate, and go by the request of op opinions for
and against and you would save money and time by not doing any filtration
whatsoever and you have better airflow with minimum risk to the
environment.  But yeah I People would smell it and wondering if you’re
making meth

**Comment by ttsgeb** [comment]
To be clear, the "definitely fine" bit was completely sarcasm.

CFM ratings on fans are relatively meaningless without indications of pressures involved.  The inline fan can move a lot of CFM in a world where there are no restrictions on flow, but the use in the laser systems results in substantial flow restrictions so the fan can't reach its advertised numbers.  Adding filtration complicates that further.

A potential solution would be to include several suction fans on a fume coffin that is designed to equally disperse flow across all of them.  Effectively, you'd want to measure CFM of the shipped system and then build your filtration such that measured CFM through it and all related piping exceeds the original system.  This enables you to successfully evacuate the laser to design spec while still leaving overhead for degradation of filter media.

Another data point is that the system the laser I most recently installed shipped with is advertised as to fully exchange the enclosed volume every 5-10 seconds.  This is almost adequate, so seeking to exchange every 2-5 seconds would probably be adequate.  Assuming the vented enclosure has a volume of 60 sqft, you would want to aim to be *filtering* >1200cfm of air through your exhaust.  Filtering less than 400cfm would likely be the limit where it becomes actively inadequate.  When considering the restrictions added by the ductwork, two of the Vevor units linked above would likely be around the minimum for adequate exhaust filtration. (I am not a flow engineer, this is not legal advice)

**Comment by heaventwig** [comment]
[This vevor scrubber](https://www.vevor.com/air-scrubber-c_12068/vevor-air-scrubber-with-3-stage-filtration-stackable-negative-air-machine-800-cfm-air-cleaner-with-merv10-carbon-h13-hepa-for-home-industrial-and-commercial-use-p_010226230633), also linked in [my above comment](https://github.com/LaboratoryB/camfive/issues/1#issuecomment-2368545286), around which I designed the sequence described in [this above comment](https://github.com/LaboratoryB/camfive/issues/1#issuecomment-2368613502), is rated by VEVOR at 800 CFM *with its manufacturer-supplied filters in place*.

The camfive laser is specced for just under 489 CFM. 800 CFM should suffice.

With added pressure differentials from the presence of the laser enclosure (restricting inflow) and downstream exhaust venting ducts (restricting outflow), it might not sustain 800 CFM. It's conceivable it could drop below 500 CFM. That's why my design in the above-referenced comments includes an [inline VEVOR duct fan](https://www.vevor.com/inline-duct-fan-c_11974/vevor-4-inch-inline-duct-fan-hvac-exhaust-blower-kit-ec-variable-speed-control-p_010532060135) in the PVC sequence in the rafters beside the garage door opener (placed just so because rafters, power supply, and PVC ducting all line up in that spot). See point 4 in the first comment.

If that's all insufficient, the simplest fix I see is to use two of the scrubbers, swap the pre-filter union (upsizing a pair of 6" exhaust ducts to the 15.5" intake) for a post-filter union (downsizing a pair of 8.5" exhaust ports to a single 4" duct; keeping the metal screens in the rig where we still need to upsize the flexible 6" aluminum ducts to the 15.5" air intake on the scrubbers), and get 1600 CFM of filtered exhaust vent fans feeding into the exhaust ducts. Seems excessive, but we do like clean air. I suspect the inline duct fan in the rafters would remain essential, in this configuration, for helping all the pressure coming out of those scrubbers find its way down the vents and out to the exhaust port on the side of the house.

**Comment by ttsgeb** [comment]
I understand this. The reason I said two of the scrubbers is because in looking deeper into it, I found comments from Vevor that the scrubbers are actually rated as "400-800" cfm, so the 800 number is absolute ideal conditions with no added restrictions and the 400 number is likely poor conditions with no added restrictions.  Considering there will be added restrictions and I already have experience with items from this company rated in excess of 800cfm not effectively removing the exhaust from the machine, I highly doubt that a single scrubber as linked, even with an added inline fan, will be suitable.

**Comment by heaventwig** [comment]
Vevor fans ordered — one for inline ducting in rafters, one for filtration of exhaust from machine. Expected delivery by 3 October 2024.

**Comment by heaventwig** [comment]
@ttsgeb how are your Vevor exhaust fans doing with the heat of the air coming out of the cutters? And what's the ducting that you use — metal or pvc?

**Comment by heaventwig** [comment]
(Also @ttsgeb — have you measured that exhaust temp anywhere?)

**Comment by heaventwig** [comment]
Relatedly, this piece of the in-built exhaust ventilation system does not appear to be metallic.
![image](https://github.com/user-attachments/assets/2ef0cb95-33f5-47e3-8e9b-f5588582e837)

We could maybe also check inside it for a metal lining and, especially if it's not metal, evidence of melting (eg due to heated exhaust).

**Comment by Hurricos** [comment]
TL;DR at 400cfm with a 130W laser continuous, and generating say 500W from burning material, you should generate a ${\Delta}T = 3.3 \degree \textrm{C}$.

If we ignore that the density of air varies with temperature: at STP, 1m^3 of air is about 1kg. The specific heat of air at the same temperature is about 1kJ/kg*degK. Let's say we move 400cfm =~ $11.3m^3/min$ =~ $0.189m^3/s$. So at 630W continuous laser power + combustion:

630 / 0.19 m^3/s / 1000 J / kg * degK * 1 kg/m^3 = $3.3\degree\textrm{C}$.

Other errata:
- I'm sure we're moving more than 100cfm of air (lower average deltaT)
- I'm sure we're combusting material and adding laser thermals (more thermal energy)
- I'm sure the density of air decreases with temperature (less thermal medium per unit time)

Factors seen and discarded:
- The specific heat of air varies (in particular, increases) with temperature, but not within the bounds we care about (0-100*C); [cite](https://www.engineeringtoolbox.com/air-specific-heat-capacity-d_705.html).

(Edit: obviously there'll still be particulate condensed and cooled from the flame, but unless you expect to be unable to control the fire at the laser-point, you don't have to deal with temperature, only particulate.)

**Comment by heaventwig** [comment]
New ducting plan:
- no junction with existing ductwork
- run exhaust duct to soffit vent instead
- install soffit vent in NE corner of garage

Might want to return 4" Vevor in-line duct fan and replace with 6" model, need to [ ] check tolerances on available soffit vents vs clearance at intended installation site.

**Comment by ttsgeb** [comment]
I have never actually noticed any issues with exhaust temperatures.  It is almost exclusively smoke and resin vapors that cool substantially even by the time they are evacuated from the machine.  The main issue with heat is going to happen when there's enough associated mass to cause a problem. Where you might see this is if embers from cutting cardboard make it through the exhaust system, or exceptionally well heated small bits of wood.  The real thing to watch is to ensure that your hose doesn't fill with debris or become so heavily coated in residue that the non-hose materials inside the hose offer fuel that can be ignited by an ember.

**Comment by heaventwig** [comment]
Ducting materials mostly acquired. Not pictured: a roll of pro-grade foil tape.
![image](https://github.com/user-attachments/assets/05f507d6-14d5-4687-a941-7de397cdaaa8)
![image](https://github.com/user-attachments/assets/1b165c4b-0261-4488-8768-f23e30020c0d)
![image](https://github.com/user-attachments/assets/ca276392-da7d-4ddf-aa54-2a2b78cf0f7d)

Still need
- [ ] reducer/expander connection from the 8" T out to the 12" collar, possibly in two steps
- [ ] another 90° bend at 8", 12", or maybe 10", to go somewhere between the 8" T and the 12" collar
- [ ] crimpers
- [ ] some kind of gasket sealant or chimney insulation for the joint between the 12" collar and the Vevor intake

Getting the crimpers because the number of (still returnable, unused) connectors in this run makes the price of a pair of crimpers desirable by comparison.

**Comment by DaBlender** [comment]
![9130A2D4-8DA5-46BC-97DD-EBCBBB57F2B6](https://github.com/user-attachments/assets/b06c1086-e6a1-4362-b00b-bf111e17292b)
